[2024-04-25T20:53:07.389+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-25T20:53:07.479+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_dag.task_1 manual__2024-04-25T20:49:06.044931+00:00 [queued]>
[2024-04-25T20:53:07.498+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_dag.task_1 manual__2024-04-25T20:49:06.044931+00:00 [queued]>
[2024-04-25T20:53:07.499+0000] {taskinstance.py:2303} INFO - Starting attempt 2 of 5
[2024-04-25T20:53:07.556+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): task_1> on 2024-04-25 20:49:06.044931+00:00
[2024-04-25T20:53:07.592+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=1064) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-04-25T20:53:07.598+0000] {standard_task_runner.py:63} INFO - Started process 1076 to run task
[2024-04-25T20:53:07.602+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'task_1', 'manual__2024-04-25T20:49:06.044931+00:00', '--job-id', '110', '--raw', '--subdir', 'DAGS_FOLDER/python_op.py', '--cfg-path', '/tmp/tmpcti848wd']
[2024-04-25T20:53:07.614+0000] {standard_task_runner.py:91} INFO - Job 110: Subtask task_1
[2024-04-25T20:53:09.473+0000] {task_command.py:426} INFO - Running <TaskInstance: my_dag.task_1 manual__2024-04-25T20:49:06.044931+00:00 [running]> on host f9a12f9b65d4
[2024-04-25T20:53:10.224+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='my_dag' AIRFLOW_CTX_TASK_ID='task_1' AIRFLOW_CTX_EXECUTION_DATE='2024-04-25T20:49:06.044931+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-25T20:49:06.044931+00:00'
[2024-04-25T20:53:10.240+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-25T20:53:10.293+0000] {logging_mixin.py:188} INFO - 1
[2024-04-25T20:53:10.304+0000] {logging_mixin.py:188} INFO - test
[2024-04-25T20:53:10.305+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-04-25T20:53:10.308+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-25T20:53:10.350+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=task_1, execution_date=20240425T204906, start_date=20240425T205307, end_date=20240425T205310
[2024-04-25T20:53:10.497+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-25T20:53:10.658+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-25T20:53:10.668+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
